build/GameData/Kopernicus/Config/SolarPanels.cfg

```diff
@@ -1,10 +1,12 @@
-@PART:FINAL
+@PART[*]:HAS[@MODULE[ModuleDeployableSolarPanel]]:FINAL
```

Comment:
I would also remove the `[*]` after `PART`

I don't think it adds anything to have it there except for (maybe?) triggering an unnecessary check

Reply:
also, I don't know the difference between

```
@Kopernicus:HAS[@MODULE[ModuleDeployableSolarPanel]]:FINAL
```
and
```
@Kopernicus:HAS[@MODULE:HAS[#name[ModuleDeployableSolarPanel]]]:FINAL
```

I usually go for the latter, but that's only because it avoided some very old MM bugs that have probably been patched by now. However it has been a long time since I tested it so 🤷‍♂